cypress-custom/integration/swap.test.ts

```diff
@@ -42,6 +42,9 @@ describe('Swap (custom)', () => {
     cy.get('#swap-currency-input .token-amount-input').should('be.visible')
     cy.get('#swap-currency-input .token-amount-input').type('0.05', { force: true, delay: 400 })
     cy.get('#swap-currency-output .token-amount-input').should('not.equal', '')
+    cy.get('#fees-exceed-checkbox').then((el) => {
+      if (el) el.click()
```

Comment:
What about

```suggestion
      el?.click()
```

Reply:
Also, can't you instead of `.then()` do a `.click()`?

Reply:
Unfortunately I can't, because the `#fees-exceed-checkbox` element availability is optional

Reply:
~But, don't we control when it's visible or not?~
Oh, does this depend on the answer from the API? Then yeah, I see your point